ext/IJuliaExt.jl

```diff
@@ -1,10 +1,15 @@
-const _use_local_dependencies = Ref(false)
-const _use_local_plotlyjs = Ref(false)
+module IJuliaExt
```

Comment:
Did you test, that this works in jupyter notebooks?

Reply:
Good point, I haven't used the Ijulia kernel in notebooks for a long time but this must be checked. Will do.

Reply:
![IJulia](https://user-images.githubusercontent.com/13423344/218417689-ff525c33-731a-4687-8aef-f47a4ce33dfa.png)

Looks like it's working. Should I check something else (I'm not familiar with IJulia) ?

Reply:
thats fine